### ISSUE TITLE
Animation issue, when a11y module loaded and marker disabled.

### ISSUE THREAD

**Issue by jakubjanuchta** [highcharts/highcharts#19117]
#### Expected behaviour
When loading a11y module and the markers are disabled, the marker should only be visible and animated on the currently hovered point so as it does when a11y not loaded.

#### Actual behaviour
When loading a11y module and the markers are disabled, the points that were hovered previously have a short fade out animation, what doesn't happen when a11y not loaded.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/kgaLqwre/

1) Hover over couple of points, one after another.
2) Notice, how the points, that were hovered have a fade out animation.
3) Comment out the a11y module.
4) Hover once again.
5) Only the marker of the currently hovered point is visible.

#### Product version
Highcharts v11

#### Affected browser(s)
Chrome

**Comment by jakubjanuchta** [comment]
Workaround: https://jsfiddle.net/BlackLabel/9xbcheqd/

**Comment by karolkolodziej** [comment]
FYI @oysteinmoseng
